package/scripts/postinstall

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# verify the files exist
+tmp_brew="$2"
+if [[ ! -d "${tmp_brew:?}" ]]; then
+    echo "no directory at $tmp_brew, exiting"
+    exit 1
+fi
+
+# pick the correct target
+if [[ "$3" != "/" ]]; then
+  target=$3
+elif [[ $(uname -m) == "x86_64" ]];then
+  target="/usr/local"
+else
+  target="/opt/Homebrew"
+fi
+
+loggedInUser=$(echo "show State:/Users/ConsoleUser" | scutil | awk '/Name :/ { print $3 }')
+if [[ -f "$target/bin/brew" ]]; then
+    if [[ $(sudo -u"$loggedInUser" "$target/bin/brew" developer) =~ "enabled" ]]; then
+      echo "developer install, exiting"
+      exit 0
+    fi
+    if [[ $("$tmp_brew/bin/brew" --version | head -n1) != $("$target/bin/brew" --version | head -n1) ]]; then
+      echo "already an outdated install at $target, updating"
+      sudo -u"$loggedInUser" "$target/bin/brew" update --auto-update
```

Comment:
Really minor, but when this ships it might be worth teaching `brew update` to update `pkginfo` information if present.